client/js/components/GameView/MoveChoiceOverlay.vue

```diff
@@ -245,6 +245,9 @@ export default {
       });
     },
     cardWidth() {
+      if (this.$vuetify.breakpoint.name === 'xs') {
```

Comment:
More concise alternative ([docs](https://vuetifyjs.com/en/features/breakpoints/#breakpoint-service-object))
```suggestion
      if (this.$vuetify.breakpoint.xs) {
```